src/cpp/core/r_util/RSourceIndex.cpp

```diff
@@ -622,7 +705,7 @@ RSourceIndex::RSourceIndex(const std::string& context, const std::string& code)
 
    // tokenize and create token cursor
    std::wstring wCode = string_utils::utf8ToWide(code, context);
-   RTokens rTokens(wCode, RTokens::StripWhitespace | RTokens::StripComments);
+   RTokens rTokens(wCode, RTokens::StripWhitespace);
```

Comment:
(It would probably suffice to make sure we're using routines like `moveToNextSignificantToken()` instead of `moveToNextToken()` in cases we want to skip whitespace / comments)

Reply:
I think I updated them now, i.e. these do show up correctly: 

```r
setMethod("plot", c("track1", "missing"),
          function(x,  y, ...) plot(x@x, x@y, ...)
)
setMethod("plot", # comment
          c("track2", "missing"),
          function(x,  y, ...) plot(x@x, x@y, ...)
)
setMethod("plot", # comment
          c(# comment
            "track3",
            "missing"),
          function(x,  y, ...) plot(x@x, x@y, ...)
)

setMethod("plot", signature(x="track4", # comment
                            y="missing"),
          function(x,  y, ...) plot(x@x, x@y, ...)
)

fun1 <- function() # comment
  42

Queue <- R6Class("Queue",
                 public = list(
                   initialize = function(...) {
                     for (item in list(...)) {
                       self$add(item)
                     }
                   },
                   add = function(x) {
                     private$queue <- c(private$queue, list(x))
                     invisible(self)
                   },
                   remove = function() {
                     if (private$length() == 0) return(NULL)
                     # Can use private$queue for explicit access
                     head <- private$queue[[1]]
                     private$queue <- private$queue[-1]
                     head
                   }
                 ),
                 private = list(
                   queue = list(),
                   length = function() base::length(private$queue)
                 )
)
```